### PR TITLE
docs: minor improvements to docs (#141)

### PR DESCRIPTION
**Pull request by johanbook**

### CODE REVIEW

File: docs/docs/client-side/cookieless.md [hunk 1]
```diff
@@ -36,15 +41,16 @@ compliance in using Dreamdata as a processor.
 
 ## How to set it up
```
Comment (LiKang6688) [inline docs/docs/client-side/cookieless.md:42]:
```suggestion
## How to set it up?
```

**Review by agjs — APPROVED** [review]

File: docs/docs/client-side/cookieless.md [hunk 2]
```diff
@@ -36,15 +41,16 @@ compliance in using Dreamdata as a processor.
...
 
-## Integrating with popular CMPs
+## Integrating with popular Consent Management Platforms
 
-Our Cookieless Analytics should work out of the box with most popular CMPs. If
-your CMP is not listed here, contact us at `ag@dreamdata.io` to see if we can
-add it.
+Cookieless Account Analytics works with many popular Consent Management
+Platforms. If your Consent Management Platform is not listed here, contact us at
+`ag@dreamdata.io` to see if we can add it.
```
Comment (LiKang6688) [inline docs/docs/client-side/cookieless.md:53]:
```suggestion
`ag@dreamdata.io` to see if we can add support to it.
```

Comment (johanbook) [reply docs/docs/client-side/cookieless.md]: Good catch. Updated.

File: docs/docs/client-side/cookieless.md (hunk 1 above)
Comment (johanbook) [reply docs/docs/client-side/cookieless.md:42]: I am thinking the topic is how to do it, rather than asking the question ☺️

**Review by LiKang6688 — APPROVED** [review]

**Pull request merged**